packages/react-aria-components/docs/TimeField.mdx

```diff
@@ -277,7 +277,9 @@ The states, selectors, and render props for each component used in a `TimeField`
 
 ### TimeField
 
-A `TimeField` can be targeted with the `.react-aria-TimeField` CSS selector, or by overriding with a custom `className`. It provides a <TypeLink links={statelyDocs.links} type={statelyDocs.exports.DateFieldState} /> object to its render props, which can be used to customize the `className`, `style`, or `children`.
+A `TimeField` can be targeted with the `.react-aria-TimeField` CSS selector, or by overriding with a custom `className`. It supports the following states and render props:
```

Comment:
```suggestion
A `TimeField` can be targeted with the `.react-aria-TimeField` CSS selector, or by overriding with a custom `className`. It supports the following states:
```
TextField called `isDisabled` and `validationState ` states, and this one has those two and `state` so I think we should remove render props here.

Reply:
Since those are render props, I'm fine keeping the text. We have it both ways throughout different examples, and both are right. I didn't change them all in order to keep the PR simple.